.github/workflows/commit-check.yml

```
@@ -27,19 +27,28 @@ jobs:
         run: |
           set +e
 
-          output=$(./_assets/scripts/commit_check.sh 2>&1)          
+          output=$(./_assets/scripts/commit_check.sh 2>&1)
           exit_code=$?
+          
+          echo "$output" | sed '$d'
+          echo "has_breaking_changes=$has_breaking_changes"
```

tiny nitpick: SC2086 using {} around variable names in ""
```suggestion
          echo "has_breaking_changes=${has_breaking_changes}"
```

---

Fixed: 56213cc77cfd3c5b3aa362086f04c3a07e86dcaf